src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -167,6 +174,14 @@ public string[] ImageArray
             }
             set
             {
+                if (defaultLottieView != null)
+                {
+                    defaultLottieView.Stop();
+                    defaultLottieView.Dispose();
+                    defaultLottieView = null;
+                }
```

Comment:
null checking can be removed
```
                    defaultLottieView?.Stop();
                    defaultLottieView?.Dispose();
                    defaultLottieView = null;
```

Reply:
Updated the code. Thanks for your review!